_source/_posts/2021-06-01-kubernetes-spring-boot-jhipster.adoc

```diff
@@ -764,7 +764,7 @@ Retrieve the certificate keypair that this controller generates.
 kubectl get secret -n kube-system -l sealedsecrets.bitnami.com/sealed-secrets-key -o yaml
 ----
 
-Copy the raw value of `tls.crt` and decode it. You can use the command line or an https://www.base64decode.org/[online base64 decoder].
+Copy the raw value of `tls.crt` and decode it. You can use the command line, or learn more about https://developer.okta.com/docs/guides/implement-grant-type/clientcreds/main/#base64-encode-the-client-id-and-client-secret[base64 encoding/decoding] in our documentation.
```

Comment:
It seems like a better link would be https://developer.okta.com/docs/guides/implement-grant-type/clientcreds/main/#base64-encode-the-client-id-and-client-secret. For some reason, this URL redirects to https://developer.okta.com/docs/guides/implement-grant-type/clientcreds/main/#flow-specifics. This seems like a bug in our developer docs.

Reply:
I opened: https://github.com/oktadev/okta-blog/issues/977 to track this